bundles/org.openhab.binding.solax/src/main/resources/OH-INF/thing/localConnectInverter.xml

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="solax"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="local-connect-inverter">
+
+		<label>Local Connect Inverter</label>
+		<description>The inverter representation that supports local connections via HTTP</description>
+
+		<channels>
+			<channel id="inverter-output-power" typeId="system.electric-power">
+				<label>Inverter Input/Output Power</label>
+				<description>Power to/from the inverter</description>
+			</channel>
+			<channel id="inverter-current" typeId="system.electric-current">
+				<label>Inverter Input/Output Current</label>
+				<description>Current to/from the inverter</description>
+			</channel>
+			<channel id="inverter-voltage" typeId="system.electric-voltage">
+				<label>Inverter Voltage</label>
+				<description>Voltage of the inverter</description>
+			</channel>
+			<channel id="inverter-frequency" typeId="frequency">
+				<label>Inverter Frequency</label>
+				<description>Frequency of the electricity to/from the inverter</description>
```

Comment:
This overrides the label/description of the channel type and is not consistent with the current label/description of the channel type. You can remove them from here and instead correct the label/description of the channel type.